### DISCUSSION TITLE
Where are osu!argon skins (Lazer default skins) objects located?

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29525] [Q&A] [answered]
I heard that osu!argon skins files are coded into the game, and thats the reason that when you try to export the skin, or a edited version of it, you only get json files, but don't get any of the icons and other files that the skin uses.
I personaly like the argon skin and i'd like to make some slight coloring variations to it, but i cant find any of the skins files and i don't want to bother developers about making the skins objects hue/overall colour adjustable in game, if you can just use a graphics program and edit the files yourself.
If anyone knows where they are located in the source code, or any other place where they are - i'd be very thankful if you could tell me where are the files.

**Comment by bdach** [comment, marked as answer]
There are no *sprites* for argon skin. The argon skin hitcircles are drawn on the fly by the game. Their appearance is described in source code.

**Reply by Tenexxt** [reply]
You said that the hitcircles are drawn by the game, but does that include the argon-wedge and counters like combo, accuracy ect.?

**Reply by bdach** [reply]
> the argon-wedge 

Yes.

> counters like combo, accuracy ect.?

These are actually sprites. See https://github.com/ppy/osu-resources/tree/master/osu.Game.Resources/Textures/Gameplay/Fonts.
